setuptools/config/expand.py

```diff
@@ -51,7 +51,7 @@
     from setuptools.dist import Distribution
 
 _K = TypeVar("_K")
-_V = TypeVar("_V", covariant=True)
+_VCo = TypeVar("_VCo", covariant=True)
```

Comment:
It seems that the convention in `typeshed` is `_V_co` (example: https://github.com/python/typeshed/blob/e9b3b03764d0557e103e1b2c4d95ce02965db22b/stdlib/builtins.pyi#L79, https://github.com/python/typeshed/blob/e9b3b03764d0557e103e1b2c4d95ce02965db22b/stubs/aiofiles/aiofiles/base.pyi#L8)... Not that I care much about it, but if we are not following a convention that I prefer the shorter `_V`...

Reply:
You are correct. Ironically I copied this from the typeshed stub... where I made the same mistake and it wasn't, caught there!
![image](https://github.com/user-attachments/assets/e6813c6c-d73e-4a4d-9c3b-145a6f3ae00d)